TestWebService/Sources/TestWebService/TestWebService.swift

```diff
@@ -47,5 +48,9 @@ struct TestWebService: Apodini.WebService {
         }
         
         WebSocket()
+        
+        Migrator(
```

Comment:
One thing that is currently missed as far as I can see is taking a look at the special case when a web service contains `@Arguments` or `@Options` from the argument parser. 

Similar to the #343 and #341 ... it would make sense to test this in here and maybe also in the example that the subcommand also works when you have arguments/options or additional things from the Swift Argument Parser in the main WebService.

I think you can take inspiration from #343 and #341 and the comments in there on how to address this in the subcommands.

Reply:
Solved it as done in #341